backend/planty/settings/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
```

Comment:
Można tutaj dodać receivery które będą tworzyły/aktualizowały Settings po aktualizacji usera.
Tutaj jest to wyjaśnione: https://simpleisbetterthancomplex.com/tutorial/2016/07/22/how-to-extend-django-user-model.html#onetoone